cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-18 06:06
+
+import cinema.utils
```

Comment:
Ensure that the `cinema.utils` module contains the `movie_image_path` function. This function is used in the `upload_to` parameter of the `ImageField`. If it's not defined, the migration will fail.